src/epivizFileParser/BamFile.py

```diff
@@ -80,6 +80,13 @@ def getRange(self, chr, start, end, bins=2000, zoomlvl=-1, metric="AVG", respTyp
 
             chrTemp = startTemp = endTemp = valueTemp = None
             for x in iter:
+
+                if not x.reference_pos >= start or not x.reference_pos <= end:
+                        continue
+                
+                if x.reference_pos == end and valueTemp is not None:
+                    result.append((chrTemp, startTemp, end, valueTemp))
```

Comment:
This would potentially add the last interval twice... needs a `continue` here

Reply:
it's weird how this never caused any issue in testing

but I added it since it makes sense to